eng/scripts/Get-BinarySizes.ps1

```diff
@@ -41,7 +41,7 @@ function setEnvVar($key, $value) {
 }
 
 function getTargetOs {
-    if ($OsVMImage.StartsWith('macOS', $true, (Get-Culture).InvariantCulture)) {
+    if ($OsVMImage -like 'macOS') {
```

Comment:
It needs to be `macOS*`.

Reply:
It's not a regex but rather is glob-like in its evaluation behavior.

Reply:
Yeah, I just figured that out :(. Cargo Cult Typing at its finest (and sloppy reading).